common/lib/user.test.js

```diff
@@ -219,6 +219,20 @@ describe('User class', function() {
       })
     })
 
+    context('when user has ROLE_PECS_PMU', function() {
+      beforeEach(function() {
+        permissions = user.getPermissions(['ROLE_PECS_PMU'])
+      })
+
+      it('should contain correct permission', function() {
+        expect(permissions).to.deep.equal([
```

Comment:
Maybe it's an idea to make a helper function that sorts the arrays so that we don't have to worry about the order...

Reply:
In the test or in the library?

I realised we were missing this test so have added it as part of this change.